recipes/snappy/bld.bat

```diff
@@ -0,0 +1,12 @@
+cd %SRC_DIR%
+git clone https://github.com/kmanley/snappy-msvc
```

Comment:
Good point.

> Can we clone a specific hash?

Like this. Maybe with a shallow clone too?

> PS: Maybe we should just download those files and ship with the recipe. Just adding a comment about that repo.

Not sure how I feel about this.

So, I'm almost leaning to having `snappy-msvc` broken out as its own package that just does the git clone with a sha. It feels cleaner IMHO. Maybe others have thoughts about this? @pelson?